data/net/minecraft/data/registries/RegistriesDatapackGenerator.mapping

```diff
@@ -2,3 +2,9 @@ CLASS net/minecraft/data/registries/RegistriesDatapackGenerator
 	METHOD <init> (Lnet/minecraft/data/PackOutput;Ljava/util/concurrent/CompletableFuture;)V
 		ARG 1 output
 		ARG 2 registries
+	METHOD dumpRegistryCap (Lnet/minecraft/data/CachedOutput;Lnet/minecraft/core/HolderLookup$Provider;Lcom/mojang/serialization/DynamicOps;Lnet/minecraft/resources/RegistryDataLoader$RegistryData;)Ljava/util/Optional;
+		ARG 1 output
+		ARG 2 registries
+	METHOD dumpValue (Ljava/nio/file/Path;Lnet/minecraft/data/CachedOutput;Lcom/mojang/serialization/DynamicOps;Lcom/mojang/serialization/Encoder;Ljava/lang/Object;)Ljava/util/concurrent/CompletableFuture;
+		ARG 0 valueOutput
```

Comment:
It may just be work calling this `valuePath` since output in both contexts is a bit confusing.